paas_app_charmer/flask/charm.py

```diff
@@ -43,13 +30,19 @@ class FlaskConfig(BaseModel, extra=Extra.allow):  # pylint: disable=too-few-publ
             context in the Flask application.
     """
 
-    env: str | None = Field(default=None, min_length=1)
-    debug: bool | None = Field(default=None)
-    secret_key: str | None = Field(default=None, min_length=1)
-    permanent_session_lifetime: int | None = Field(default=None, gt=0)
-    application_root: str | None = Field(default=None, min_length=1)
-    session_cookie_secure: bool | None = Field(default=None)
-    preferred_url_scheme: str | None = Field(default=None, pattern="(?i)^(HTTP|HTTPS)$")
+    env: str | None = Field(alias="flask-env", default=None, min_length=1)
+    debug: bool | None = Field(alias="flask-debug", default=None)
+    secret_key: str | None = Field(alias="flask-secret-key", default=None, min_length=1)
+    permanent_session_lifetime: int | None = Field(
+        alias="flask-permanent-session-lifetime", default=None, gt=0
+    )
+    application_root: str | None = Field(
+        alias="flask-application-root", default=None, min_length=1
+    )
+    session_cookie_secure: bool | None = Field(alias="flask-session-cookie-secure", default=None)
+    preferred_url_scheme: str | None = Field(
```

Comment:
Isn't this simpler with a literal?

Reply:
Oh! thanks for the tip, TIL!

Reviewing it, if you agree, I will leave the regex as it handles case insensitivity.

Reply:
Sure thing